cdc/cdc/sink/tikv.go

```diff
@@ -204,12 +206,36 @@ type innerBatch struct {
 	OpType model.OpType
 	Keys   [][]byte
 	Values [][]byte
+	TTL    []uint64
+
+	count int
 }
 
 type tikvBatcher struct {
-	Batches  map[model.OpType]*innerBatch
-	count    int
-	byteSize int64
+	Batches        map[model.OpType]*innerBatch
+	BatchesWithTTL map[model.OpType]*innerBatch
+	count          int
+	byteSize       int64
+}
+
+func NewTiKVBatcher() *tikvBatcher {
+	tikvBatcher := &tikvBatcher{}
+	tikvBatcher.Batches[model.OpTypePut] = &innerBatch{
```

Comment:
`TTL == 0` is treated as never expired. So we can put `Batches` & `BatchesWithTTL` together.

Reply:
TODO: fix in next PR.